core/src/main/scala/akka/kafka/ProducerSettings.scala

```diff
@@ -307,7 +307,7 @@ class ProducerSettings[K, V] @InternalApi private[kafka] (
     ")"
 
   /**
-   * Create a `KafkaProducer` instance from the settings.
+   * Create a `Producer` instance from the settings.
    */
-  def createKafkaProducer(): KafkaProducer[K, V] = producerFactory.apply(this)
+  def createProducer(): Producer[K, V] = producerFactory.apply(this)
```

Comment:
We have many things called "Producer", the method name could be kept as `createKafkaProducer` and with `Producer` as return type.

Reply:
I changed the name back.